testsuites/CBLTester/p2p_topology_specific_tests/test_p2p_multiple_sgs.py

```diff
@@ -203,4 +203,4 @@ def create_sync_gateways(cluster_config, sg_config_path):
     cluster.reset(sg_config_path=sg_config_path)
     sg1 = cluster.sync_gateways[0]
     sg2 = cluster.sync_gateways[1]
-    return sg1, sg2
\ No newline at end of file
+    return sg1, sg2
```

Comment:
the top 4 files are no change, right? if so, I would like to remove them from change list, it will shorten the change history for the file, once need to trace something, it will be easier.

Reply:
:) There is invisible space change, flake8 is complaining about all those file . These are the files that Sridevi mentioned